v1/linkml-schemas/name.yaml

```diff
@@ -17,34 +17,38 @@ default_range: string
 
 classes:
   NameClass:
+    description: Name object that represnts a user's claimed identity.  This will have a list of name parts.  Usually this will contain one or more [GivenNameClass](../GivenNameClass) name parts and one [FamilyNameClass](../FamilyNameClass) name part for a specific period of validity.
```

Comment:
```suggestion
    description: Name object that represents a user's claimed identity. This will have a list of [`nameParts`](../nameParts). Usually this will contain one or more [`GivenName`](../GivenNameClass) name parts and one (or occasionally more) [`FamilyName`](../FamilyNameClass) name part(s) for a specific period of validity.
```